src/test/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCountsIntegrationTest.java

```diff
@@ -262,46 +260,24 @@ public void testFullReadCountsInputRunOneSample() throws IOException {
                input.targets().stream().collect(Collectors.toSet()));
         assertFactorNormalizedValues(input, factorNormalized);
 
-        final TargetCollection<TargetCoverage> preTangentNormalizedAsTargetCollection = TargetCoverageUtils.readModeledTargetFileIntoTargetCollection(preTangentNormalizationOutput);
-        final ReadCountCollection preTangentNormalized = convertToReadCountCollection(preTangentNormalizedAsTargetCollection, "SAMPLE1");
+        final ReadCountCollection preTangentNormalized = ReadCountCollectionUtils.parse(preTangentNormalizationOutput);
         assertPreTangentNormalizedValues(factorNormalized, preTangentNormalized);
         assertBetaHats(preTangentNormalized, betaHats, TEST_PON);
         assertBetaHatsRobustToOutliers(preTangentNormalized, TEST_PON);
 
         // Test the tangent normalized output
-        final TargetCollection<TargetCoverage> tangentNormalizedAsTargetCollection = TargetCoverageUtils.readModeledTargetFileIntoTargetCollection(tangentNormalizationOutput);
-        final ReadCountCollection tangentNormalized = convertToReadCountCollection(tangentNormalizedAsTargetCollection, "SAMPLE1");
+        final ReadCountCollection tangentNormalized = ReadCountCollectionUtils.parse(tangentNormalizationOutput);
         assertTangentNormalized(tangentNormalized, preTangentNormalized, betaHats, TEST_PON);
         Assert.assertEquals(tangentNormalized.columnNames(), input.columnNames());
         Assert.assertEquals(tangentNormalized.columnNames(), preTangentNormalized.columnNames());
 
         // Make sure that we can read in the tangent normalized targets as a collection of TargetCoverage
-        final TargetCollection<TargetCoverage> targets = TargetCoverageUtils.readModeledTargetFileIntoTargetCollection(new File(tangentNormalizationOutput.getAbsolutePath()));
-        Assert.assertEquals(targets.targetCount(), 657);
-        Assert.assertEquals(targets.targets().get(2).getName(), "target_179700_CRYBB1");
+        Assert.assertEquals(tangentNormalized.targets().size(), 657);
+        Assert.assertEquals(tangentNormalized.targets().get(2).getName(), "target_179700_CRYBB1");
     }
 
-    @Test(expectedExceptions = UserException.BadInput.class)
```

Comment:
I replaced this test with one that is supposed to ruin without an exception since now there's no reason to forbid multiple samples.